src/ripple/app/consensus/RCLValidations.cpp

```diff
@@ -152,7 +152,8 @@ bool
 handleNewValidation(
     Application& app,
     std::shared_ptr<STValidation> const& val,
-    std::string const& source)
+    std::string const& source,
+    bool relayUntrusted)
```

Comment:
`relayUntrusted` (basically is `RELAY_UNTRUSTED_VALIDATIONS`) is used to override the return of `handleNewValidation`. I think it'll be more clear just to evaluate `RELAY_UNTRUSTED_VALIDATIONS` when deciding whether relay or not. It also simplifies the function's signature and reduces the required changes.

Reply:
Thanks, that's nice. Adopted.